instrumentation/message-handler/library/README.md

```diff
@@ -0,0 +1,30 @@
+# Message Handler
+
+This package contains handlers to instrument message system spans.
```

Comment:
May add user story that using BatchMessageHandler interface for a batch of messages fan in  case.

Reply:
Agreed.